website/Bomberjam.Website/Frontend/scripts/react/viewerWithControls.tsx

```diff
@@ -84,6 +84,9 @@ export const ViewerWithControls = (props: VisualizerProps) => {
               <button onClick={decreaseSpeed} className="btn btn-primary" disabled={!isStarted}>Slower</button>
               <button onClick={increaseSpeed} className="btn btn-primary" disabled={!isStarted}>Faster</button>
             </div>
+            <div className="btn btn-sm ml-2" style="pointer-events:none;">
```

Comment:
React.CSSProperties instead of style string
```suggestion
            <div className="btn btn-sm ml-2" style={{pointerEvents: "none"}}>
```

Reply:
Style needs to be an object?